dotnet/src/Extensions/Experimental/Orchestration.Flow/FlowOrchestrator.cs

```diff
@@ -80,7 +75,7 @@ public class FlowPlanner
             throw new SKException("Invalid flow", ex);
         }
 
-        FlowExecutor executor = new(this._kernelBuilder, this._flowStatusProvider, this._globalSkillCollection, this._config);
+        var executor = new FlowExecutor.FlowExecutor(this._kernelBuilder, this._flowStatusProvider, this._globalSkillCollection, this._config);
```

Comment:
Can we inject the executor the same way the flow validator is injected through the class constructor? It will simplify the class constructor and tests.

Reply:
`FlowExecutor` is a pretty complex module and it is quite unlikely that users of this extension would provide a extended/customized one, instead of just having another `FlowOrchestrator`.